pkg/minikube/translate/translate.go

```diff
@@ -0,0 +1,115 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package translate
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"strings"
+
+	"github.com/cloudfoundry-attic/jibber_jabber"
+	"github.com/golang/glog"
+	"golang.org/x/text/language"
+)
+
+var (
+	// preferredLanguage is the default language messages will be output in
+	preferredLanguage = language.AmericanEnglish
+	// our default language
+	defaultLanguage = language.AmericanEnglish
+
+	// Translations is a translation map from strings that can be output to console
+	// to its translation in the user's system locale.
+	Translations map[string]interface{}
+)
+
+// Translate translates the given string to the supplied language.
+func Translate(s string) string {
+	if preferredLanguage == defaultLanguage {
+		return s
+	}
+
+	if len(Translations) == 0 {
+		return s
+	}
+
+	if translation, ok := Translations[s]; ok {
```

Comment:
Use shorter variable names for shorter scopes, longer variable names for long scopes. This should perhaps be "t" instead of translation.

When I saw the next line, I assumed "translation" was a global due to its variable name length.